src/add-search-spec.js

```diff
@@ -41,25 +46,38 @@ describe('Search Results', () => {
 
   beforeEach(() => {
     jasmine.DEFAULT_TIMEOUT_INTERVAL = 500000;
-    files = walkSync(path.join(__dirname, '..', 'src', 'app'));
-    files = files.map(file => {
-      let route = file.split('/app')[1];
-      route = route.slice(0, route.length - 11);
-      if (route === '') {
-        route = '/';
-      }
-      return route;
-    });
   });
 
   it('should generate search results', (done) => {
-    let config = JSON.parse(browser.params.skyPagesConfig);
-    let siteName = config.skyux.name;
+    // Comment on why we need these separate configs for the url
+    let config: any = browser.params.skyPagesConfig;
```

Comment:
If these variables never change, would it be better to use `const` so we can be sure they don't get altered later?